src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceData/CounterSet.cs

```diff
@@ -256,7 +256,7 @@ public CounterSetInstance CreateCounterSetInstance(string instanceName)
                             {
                                 throw Status switch
                                 {
-                                    (uint)Interop.Errors.ERROR_ALREADY_EXISTS => new ArgumentException(SR.Format(SR.Perflib_Argument_CounterSetAlreadyRegister, _counterSet), "CounterSetGuid"),
+                                    (uint)Interop.Errors.ERROR_ALREADY_EXISTS => new InvalidOperationException(SR.Format(SR.Perflib_Argument_CounterSetAlreadyRegister, _counterSet)),
```

Comment:
Is this a breaking change?  That might be fine, but we should call it out as necessary.

Reply:
Yes, @bartonjs suggested to add into breaking change doc, i will after finding out how to do that